src/Controls/src/Core/RefreshView/RefreshView.cs

```diff
@@ -135,6 +112,20 @@ public IPlatformElementConfiguration<T, RefreshView> On<T>() where T : IConfigPl
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
 
+		ICommand ICommandElement.Command => Command;
+
+		object ICommandElement.CommandParameter => CommandParameter;
+
+		protected override bool IsEnabledCore => base.IsEnabledCore && CommandElement.GetCanExecute(this);
+		
+		void ICommandElement.CanExecuteChanged(object sender, EventArgs e)
+		{
+			if((bool)GetValue(IsRefreshingProperty))
+				return;
```

Comment:
I see this code here and if for some reason the can execute is about to change but the refreshing is happening, the change will not take place? What happened to make you put this here? I feel like this will do something, but also maybe you were just being safe.

I see there is some coercion happening with the IsRefreshing property too.

Maybe if we do this, we need a "mirror" item in the `OnIsRefreshingPropertyChanged` to also call the RefreshIsEnabledProperty`? Not sure what that would do or if it creates a circular system.

What happens if you set IsEnabled to false while is it in the process of refreshing? Does it stop the refresh? I know if you change the refreshing when it is enabled, the coerce on IsRefreshing will just force stop. But the inverse? I would conceptually think that setting IsEnabled will not effect the current refresh but more the _next_ refresh? Maybe I am wrong and there is a legit reason to stop the refresh? But then this code is skipping the refresh which tells me that maybe we are thinking the same?